frontend/packages/pipelines-plugin/src/components/import/pipeline/PacInfoPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { ClipboardCopy, ClipboardCopyVariant, Text } from '@patternfly/react-core';
+import { FormikValues, useFormikContext } from 'formik';
+import { Trans, useTranslation } from 'react-i18next';
+import {
+  ReadableResourcesNames,
+  Resources,
+} from '@console/dev-console/src/components/import/import-types';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { useActiveNamespace } from '@console/shared/src';
+import './PacSection.scss';
+
+const InfoPanel: React.FC = () => {
+  const { t } = useTranslation();
+  const [namespace] = useActiveNamespace();
+  const { values } = useFormikContext<FormikValues>();
+  const selectedResource = ReadableResourcesNames[values.resources as Resources];
+
+  return (
+    <FormSection>
+      <Text className="odc-pipeline-section-pac__info-panel">
+        <Trans t={t} ns="pipelines-plugin">
+          <div className="odc-pipeline-section-pac__info-text">
+            Once your Pipeline Repository is configured, in order to update your{' '}
```

Comment:
When seeing the change in `packages/pipelines-plugin/locales/en/pipelines-plugin.json` I think its easier to translate this part here when you split this a little bit. Can you just wrap the box-content in a `Trans` component and then use standard `t()` for `Internal image registry:`?